### ISSUE TITLE
Sending template push

### ISSUE THREAD

**Issue by aneemtalukder** [urbanairship/python-library#35]
Is there a way to send a template push in this library? I am referring to this example here: http://docs.urbanairship.com/api/ua.html#template-creation-and-push-example

this payload is used:
```
{
    "device_types": "all",
    "audience": {
       "ios_channel": "b8f9b663-0a3b-cf45-587a-be880946e881"
    },
    "merge_data": {
        "template_id": "ef34a8d9-0ad7-491c-86b0-aea74da15161",
        "substitutions": {
            "TITLE": "Ms.",
            "FIRST_NAME": "Alice",
            "LAST_NAME": "Jones"
        }
    }
}
```

However in this library 1.0.0 I can't find a way to add that "merge_data" block that contains the template information.

**Comment by pdxmele** [comment]
Hi there! Sorry, this is not yet supported. We'll try to get it into the next release, or we welcome PRs with a signed contribution agreement. Thanks!

**Comment by pdxmele** [comment]
This is now available in 3.0.0
